website/static/js/fangorn.js

```diff
@@ -833,6 +836,45 @@ function expandStateLoad(item) {
     reapplyTooltips();
 }
 
+/**
+ * @param tree A Treebeard _item object for the row
+ * @param nodeID Current node._id
+ * @param file window.contextVars.file object
+ */
+function findCurrentFileID(tree, nodeID, file) {
```

Comment:
@sloria would `getCurrentFileID` be a more appropriate name? I'm not updating `folderIndex`, just retrieving it so that the scroll location can be calculated. The update folders is for the purpose of getting the file id (if it is in a subfolder whose data has not been loaded yet).